### DISCUSSION TITLE
Why generate_project.bat doesn't have .solution files after I run it

### DISCUSSION THREAD

**Discussion by AndreiB200** [NVIDIA-Omniverse/PhysX#109] [Q&A] [answered]
I pressed 2 for the solution to have for the Visual Studio 2022 but the .sln files are not present, what I am missing ?
![image](https://user-images.githubusercontent.com/25882953/222870060-4f27cb7d-e7b7-4daa-9cf7-23e0ca6a200d.png)

**Comment by ayoub-belarbi** [comment]
Hello @AndreiB200, can you share the generate_projects.bat's output? 
Thanks

**Reply by AndreiB200** [reply]
![image](https://user-images.githubusercontent.com/25882953/223197025-641dbd5b-d695-4876-8731-5e4cf14d5647.png)
There is nothing on the "compiler->vs17win64" and the console output from "generate_projects.bat" is closing after installation, I was watching on the internet an saw that I only needed to click on generate_projects.bat and that's all. Also I have VS2022.

**Reply by ayoub-belarbi** [reply]
The fact that compiler->vs17win64 is empty could mean that something went wrong when generating the project.
The console output is needed in order to investigate the issue. Can you open the command prompt, cd to the PhysX directory and then run the script using `.\generate_projects.bat`, then please share the output. Thanks

**Reply by AndreiB200** [reply]
![image](https://user-images.githubusercontent.com/25882953/223200379-f561aa05-15b6-41f6-99d4-708cf59e0545.png)

**Comment by ayoub-belarbi** [comment, marked as answer]
Thanks. You don't have cmake installed, which is a requirement. Please take a look at: https://github.com/NVIDIA-Omniverse/PhysX/blob/release/104.2/physx/documentation/platformreadme/windows/README_WINDOWS.md

**Reply by AndreiB200** [reply]
I already got 3.24.1 Cmake version, but for some reason now I reinstalled for a new version and works now. Thanks a lot !!
